src/lib/commands/drivers/package-util.ts

```diff
@@ -0,0 +1,121 @@
+import fs from 'fs'
+
+import { CLIError } from '@oclif/errors'
+import JSZip from 'jszip'
+import picomatch from 'picomatch'
+
+import { findYAMLFilename, isDir, isFile, readYAMLFile, requireDir } from '../../file-util'
+
+
+// Utility methods specific to the `edge:drivers:package` command. Split out here to make
+// unit testing easier.
+
+export interface YAMLFileData {
+	[key: string]: string
+}
+
+export const getProjectDirectory = (projectDirectoryFromArgs: string): string => {
```

Comment:
I actually pondered that name a lot, but not that reason you did. 😄  I don't like the `get` part of it since it's really  determining the name to use and removing a trailing slash if necessary. Perhaps the function is overloaded but it's so small already.

If I renamed it though (which I'm not opposed to, since it is the project directory name we are dealing with after all), I would want to rename the parameter as well to `projectDirNameFromArgs` since that parameter is returned exactly as the output if it's defined and doesn't end in a slash. That is to say, this input and the output _are_ the same with the exception of the input being the optional one from the args and the output is the (potentially) cleaned-up or default version to use.

Reply:
I did rename both the parameter and the function as well as the name of the internal variable that is used for the return value to include `name`.

The more I look at it, though, the more I just generally dislike this function. It also asserts the directory exists so it's really doing three different things which is why it's so hard to name.

Reply:
Maybe I should have called it `processProjectDirName`?

Reply:
Or print it out and burn it. 😁

Reply:
After taking a second look, `resolve` may be a good verb to use in place of `get`. But I think it is fine as refactored.

Reply:
I like `resolve` much better than `get`. I've used this and fixed the couple places where the old name was part of another variable name (in tests) that I forgot to fix with my previous rename.